test/runner/requirements/units.txt

```diff
@@ -21,6 +21,8 @@ netaddr
 ipaddress
 netapp-lib
 solidfire-sdk-python
+azure-mgmt ; python_version >= '2.7'
+azure-mgmt-netapp ; python_version >= '2.7'
```

Comment:
Instead of adding requirements for the unit tests, mock any necessary dependencies so you're only testing the module code and not the Azure SDK as well.